2021-06-02 20:15:24 INFO run.py:219 GNS3 server version 2.2.21
2021-06-02 20:15:24 INFO run.py:221 Copyright (c) 2007-2021 GNS3 Technologies Inc.
2021-06-02 20:15:24 INFO run.py:224 Config file /home/saad/.config/GNS3/2.2/gns3_server.conf loaded
2021-06-02 20:15:24 WARNING run.py:230 Local mode is enabled. Beware, clients will have full control on your filesystem
2021-06-02 20:15:24 INFO run.py:243 Running with Python 3.9.5 and has PID 43081
2021-06-02 20:15:24 INFO run.py:79 Current locale is en_US.UTF-8
2021-06-02 20:15:24 WARNING port_manager.py:92 Remote console connections are allowed
2021-06-02 20:15:24 INFO web_server.py:318 Starting server on localhost:3080
2021-06-02 20:15:24 INFO __init__.py:63 Load controller configuration file /home/saad/.config/GNS3/2.2/gns3_controller.conf
2021-06-02 20:15:24 INFO __init__.py:67 Controller is starting
2021-06-02 20:15:25 INFO compute.py:64 Create compute local
2021-06-02 20:15:25 INFO compute.py:364 Connecting to compute 'local'
2021-06-02 20:15:25 INFO web_log.py:206 127.0.0.1 [02/Jun/2021:15:15:25 +0000] "GET /v2/compute/capabilities HTTP/1.1" 200 552 "-" "Python/3.9 aiohttp/3.7.4.post0"
2021-06-02 20:15:26 INFO web_log.py:206 ::1 [02/Jun/2021:15:15:26 +0000] "GET /v2/version HTTP/1.1" 200 227 "-" "GNS3 QT Client v2.2.21"
2021-06-02 20:15:26 INFO notification_handler.py:50 New client has connected to compute WebSocket
2021-06-02 20:15:26 INFO compute.py:434 Connected to compute 'local' WebSocket 'http://localhost:3080/v2/compute/notifications/ws'
2021-06-02 20:15:26 INFO web_log.py:206 ::1 [02/Jun/2021:15:15:26 +0000] "GET /v2/version HTTP/1.1" 200 227 "-" "GNS3 QT Client v2.2.21"
2021-06-02 20:15:26 INFO notification_handler.py:73 New client has connected to controller WebSocket
2021-06-02 20:15:26 INFO web_log.py:206 ::1 [02/Jun/2021:15:15:26 +0000] "GET /v2/templates HTTP/1.1" 200 16980 "-" "GNS3 QT Client v2.2.21"
2021-06-02 20:15:26 INFO web_log.py:206 ::1 [02/Jun/2021:15:15:26 +0000] "GET /v2/computes HTTP/1.1" 200 1023 "-" "GNS3 QT Client v2.2.21"
2021-06-02 20:15:26 INFO web_log.py:206 ::1 [02/Jun/2021:15:15:26 +0000] "GET /v2/appliances?symbol_theme=Affinity-circle-blue HTTP/1.1" 200 795244 "-" "GNS3 QT Client v2.2.21"
2021-06-02 20:15:26 INFO web_log.py:206 ::1 [02/Jun/2021:15:15:26 +0000] "GET /v2/projects HTTP/1.1" 200 30568 "-" "GNS3 QT Client v2.2.21"
2021-06-02 20:15:26 INFO web_log.py:206 ::1 [02/Jun/2021:15:15:26 +0000] "GET /v2/projects HTTP/1.1" 200 30568 "-" "GNS3 QT Client v2.2.21"
2021-06-02 20:15:28 INFO project.py:78 Project bfe5b4cb-e036-44cf-9056-d048176303d9 with path '/home/saad/GNS3/projects/grand project' created
2021-06-02 20:15:28 INFO web_log.py:206 ::1 [02/Jun/2021:15:15:28 +0000] "POST /v2/compute/projects HTTP/1.1" 201 308 "-" "Python/3.9 aiohttp/3.7.4.post0"
2021-06-02 20:15:28 INFO qemu_vm.py:220 QEMU VM "R1" [7187d078-f6ea-450d-a123-6b45a338112c] has set the QEMU path to /usr/bin/qemu-system-x86_64
2021-06-02 20:15:28 INFO qemu_vm.py:622 QEMU VM "R1" [7187d078-f6ea-450d-a123-6b45a338112c]: MAC address changed to 0c:03:d9:11:2c:00
2021-06-02 20:15:28 INFO qemu_vm.py:570 QEMU VM "R1" [7187d078-f6ea-450d-a123-6b45a338112c]: number of Ethernet adapters changed to 1
2021-06-02 20:15:28 WARNING qemu_vm.py:138 Config disk: 'mtools' are not installed.
2021-06-02 20:15:28 INFO qemu_vm.py:147 QEMU VM "R1" [7187d078-f6ea-450d-a123-6b45a338112c] has been created
2021-06-02 20:15:28 INFO base_node.py:282 Qemu: R1 [7187d078-f6ea-450d-a123-6b45a338112c] created
2021-06-02 20:15:28 INFO qemu_vm.py:570 QEMU VM "R1" [7187d078-f6ea-450d-a123-6b45a338112c]: number of Ethernet adapters changed to 8
2021-06-02 20:15:28 INFO qemu_vm.py:263 QEMU VM "R1" [7187d078-f6ea-450d-a123-6b45a338112c] has set the QEMU hda_disk_image path to /home/saad/GNS3/images/QEMU/vios-adventerprisek9-m.vmdk.SPA.156-2.T
2021-06-02 20:15:28 INFO qemu_vm.py:367 QEMU VM "R1" [7187d078-f6ea-450d-a123-6b45a338112c] has set the QEMU hda disk interface to virtio
2021-06-02 20:15:28 INFO qemu_vm.py:263 QEMU VM "R1" [7187d078-f6ea-450d-a123-6b45a338112c] has set the QEMU hdb_disk_image path to /home/saad/GNS3/images/QEMU/IOSv_startup_config.img
2021-06-02 20:15:28 INFO qemu_vm.py:390 QEMU VM "R1" [7187d078-f6ea-450d-a123-6b45a338112c] has set the QEMU hdb disk interface to virtio
2021-06-02 20:15:28 INFO qemu_vm.py:413 QEMU VM "R1" [7187d078-f6ea-450d-a123-6b45a338112c] has set the QEMU hdc disk interface to ide
2021-06-02 20:15:28 INFO qemu_vm.py:436 QEMU VM "R1" [7187d078-f6ea-450d-a123-6b45a338112c] has set the QEMU hdd disk interface to ide
2021-06-02 20:15:28 INFO qemu_vm.py:622 QEMU VM "R1" [7187d078-f6ea-450d-a123-6b45a338112c]: MAC address changed to 0c:03:d9:60:de:00
2021-06-02 20:15:28 INFO qemu_vm.py:763 QEMU VM "R1" [7187d078-f6ea-450d-a123-6b45a338112c] has set the process priority to normal
2021-06-02 20:15:28 INFO qemu_vm.py:786 QEMU VM "R1" [7187d078-f6ea-450d-a123-6b45a338112c] has set the RAM to 512
2021-06-02 20:15:28 INFO web_log.py:206 127.0.0.1 [02/Jun/2021:15:15:28 +0000] "POST /v2/compute/projects/bfe5b4cb-e036-44cf-9056-d048176303d9/qemu/nodes HTTP/1.1" 201 1867 "-" "Python/3.9 aiohttp/3.7.4.post0"
2021-06-02 20:15:29 INFO qemu_vm.py:220 QEMU VM "S1" [77804b01-6eab-444d-8135-c4e4dc1553a5] has set the QEMU path to /usr/bin/qemu-system-x86_64
2021-06-02 20:15:29 INFO qemu_vm.py:622 QEMU VM "S1" [77804b01-6eab-444d-8135-c4e4dc1553a5]: MAC address changed to 0c:03:d9:53:a5:00
2021-06-02 20:15:29 INFO qemu_vm.py:570 QEMU VM "S1" [77804b01-6eab-444d-8135-c4e4dc1553a5]: number of Ethernet adapters changed to 1
2021-06-02 20:15:29 WARNING qemu_vm.py:138 Config disk: 'mtools' are not installed.
2021-06-02 20:15:29 INFO qemu_vm.py:147 QEMU VM "S1" [77804b01-6eab-444d-8135-c4e4dc1553a5] has been created
2021-06-02 20:15:29 INFO base_node.py:282 Qemu: S1 [77804b01-6eab-444d-8135-c4e4dc1553a5] created
2021-06-02 20:15:29 INFO qemu_vm.py:570 QEMU VM "S1" [77804b01-6eab-444d-8135-c4e4dc1553a5]: number of Ethernet adapters changed to 16
2021-06-02 20:15:29 INFO qemu_vm.py:263 QEMU VM "S1" [77804b01-6eab-444d-8135-c4e4dc1553a5] has set the QEMU hda_disk_image path to /home/saad/GNS3/images/QEMU/vios_l2-adventerprisek9-m.03.2017.qcow2
2021-06-02 20:15:29 INFO qemu_vm.py:367 QEMU VM "S1" [77804b01-6eab-444d-8135-c4e4dc1553a5] has set the QEMU hda disk interface to ide
2021-06-02 20:15:29 INFO qemu_vm.py:390 QEMU VM "S1" [77804b01-6eab-444d-8135-c4e4dc1553a5] has set the QEMU hdb disk interface to ide
2021-06-02 20:15:29 INFO qemu_vm.py:413 QEMU VM "S1" [77804b01-6eab-444d-8135-c4e4dc1553a5] has set the QEMU hdc disk interface to ide
2021-06-02 20:15:29 INFO qemu_vm.py:436 QEMU VM "S1" [77804b01-6eab-444d-8135-c4e4dc1553a5] has set the QEMU hdd disk interface to ide
2021-06-02 20:15:29 INFO qemu_vm.py:622 QEMU VM "S1" [77804b01-6eab-444d-8135-c4e4dc1553a5]: MAC address changed to 0c:03:d9:97:c9:00
2021-06-02 20:15:29 INFO qemu_vm.py:763 QEMU VM "S1" [77804b01-6eab-444d-8135-c4e4dc1553a5] has set the process priority to normal
2021-06-02 20:15:29 INFO qemu_vm.py:786 QEMU VM "S1" [77804b01-6eab-444d-8135-c4e4dc1553a5] has set the RAM to 768
2021-06-02 20:15:29 INFO web_log.py:206 ::1 [02/Jun/2021:15:15:29 +0000] "POST /v2/compute/projects/bfe5b4cb-e036-44cf-9056-d048176303d9/qemu/nodes HTTP/1.1" 201 1717 "-" "Python/3.9 aiohttp/3.7.4.post0"
2021-06-02 20:15:29 INFO qemu_vm.py:220 QEMU VM "Windows-7-1" [ad9d2b39-b19d-4898-bda1-dc1a91132157] has set the QEMU path to /usr/bin/qemu-system-x86_64
2021-06-02 20:15:29 INFO qemu_vm.py:622 QEMU VM "Windows-7-1" [ad9d2b39-b19d-4898-bda1-dc1a91132157]: MAC address changed to 0c:03:d9:21:57:00
2021-06-02 20:15:29 INFO qemu_vm.py:570 QEMU VM "Windows-7-1" [ad9d2b39-b19d-4898-bda1-dc1a91132157]: number of Ethernet adapters changed to 1
2021-06-02 20:15:29 WARNING qemu_vm.py:138 Config disk: 'mtools' are not installed.
2021-06-02 20:15:29 INFO qemu_vm.py:147 QEMU VM "Windows-7-1" [ad9d2b39-b19d-4898-bda1-dc1a91132157] has been created
2021-06-02 20:15:29 INFO base_node.py:282 Qemu: Windows-7-1 [ad9d2b39-b19d-4898-bda1-dc1a91132157] created
2021-06-02 20:15:29 ERROR route.py:222 Image missing error detected: /home/saad/virtual Machines/windows_7.qcow2
2021-06-02 20:15:29 INFO web_log.py:206 127.0.0.1 [02/Jun/2021:15:15:29 +0000] "POST /v2/compute/projects/bfe5b4cb-e036-44cf-9056-d048176303d9/qemu/nodes HTTP/1.1" 409 423 "-" "Python/3.9 aiohttp/3.7.4.post0"
2021-06-02 20:15:29 INFO base_node.py:331 Qemu: 'Windows-7-1' [ad9d2b39-b19d-4898-bda1-dc1a91132157]: is closing
2021-06-02 20:15:29 INFO base_node.py:331 Qemu: 'S1' [77804b01-6eab-444d-8135-c4e4dc1553a5]: is closing
2021-06-02 20:15:29 INFO base_node.py:331 Qemu: 'R1' [7187d078-f6ea-450d-a123-6b45a338112c]: is closing
2021-06-02 20:15:29 INFO project.py:361 Project bfe5b4cb-e036-44cf-9056-d048176303d9 with path '/home/saad/GNS3/projects/grand project' closed
2021-06-02 20:15:29 INFO web_log.py:206 ::1 [02/Jun/2021:15:15:29 +0000] "POST /v2/compute/projects/bfe5b4cb-e036-44cf-9056-d048176303d9/close HTTP/1.1" 204 205 "-" "Python/3.9 aiohttp/3.7.4.post0"
2021-06-02 20:15:29 INFO web_log.py:206 ::1 [02/Jun/2021:15:15:28 +0000] "POST /v2/projects/bfe5b4cb-e036-44cf-9056-d048176303d9/open HTTP/1.1" 409 311 "-" "GNS3 QT Client v2.2.21"
2021-06-02 20:15:30 INFO web_log.py:206 ::1 [02/Jun/2021:15:15:30 +0000] "GET /v2/computes HTTP/1.1" 200 1023 "-" "GNS3 QT Client v2.2.21"
2021-06-02 20:15:32 INFO web_log.py:206 ::1 [02/Jun/2021:15:15:32 +0000] "GET /v2/computes HTTP/1.1" 200 1023 "-" "GNS3 QT Client v2.2.21"
2021-06-02 20:15:42 INFO project.py:78 Project 2a56fee1-bb43-4cda-a5f5-1ab3c90677e4 with path '/home/saad/GNS3/projects/lab_for_programming' created
2021-06-02 20:15:42 INFO web_log.py:206 ::1 [02/Jun/2021:15:15:42 +0000] "POST /v2/compute/projects HTTP/1.1" 201 314 "-" "Python/3.9 aiohttp/3.7.4.post0"
2021-06-02 20:15:42 INFO nat.py:51 NAT node 'NAT1' configured to use NAT interface 'virbr0'
2021-06-02 20:15:42 INFO base_node.py:655 Starting new uBridge hypervisor localhost:35017
2021-06-02 20:15:42 INFO hypervisor.py:169 starting ubridge: ['/sbin/ubridge', '-H', 'localhost:35017']
2021-06-02 20:15:42 INFO hypervisor.py:171 logging to /home/saad/GNS3/projects/lab_for_programming/project-files/builtin/9c32e5f4-9a89-4939-82c0-22ddd64553a7/ubridge.log
2021-06-02 20:15:42 INFO hypervisor.py:179 ubridge started PID=43277
2021-06-02 20:15:42 INFO base_node.py:658 Hypervisor localhost:35017 has successfully started
2021-06-02 20:15:42 INFO ubridge_hypervisor.py:83 Connected to uBridge hypervisor on localhost:35017 after 0.1035 seconds
2021-06-02 20:15:42 INFO cloud.py:215 Cloud "NAT1" [9c32e5f4-9a89-4939-82c0-22ddd64553a7] has been created
2021-06-02 20:15:42 INFO web_log.py:206 127.0.0.1 [02/Jun/2021:15:15:42 +0000] "POST /v2/compute/projects/2a56fee1-bb43-4cda-a5f5-1ab3c90677e4/nat/nodes HTTP/1.1" 201 553 "-" "Python/3.9 aiohttp/3.7.4.post0"
2021-06-02 20:15:42 INFO iou_vm.py:73 IOU "R1" [f9b76d55-6850-4935-8f45-21e906e87c04]: assigned with application ID 1
2021-06-02 20:15:42 INFO iou_vm.py:848 IOU "R1" [f9b76d55-6850-4935-8f45-21e906e87c04]: number of Ethernet adapters changed to 2
2021-06-02 20:15:42 INFO iou_vm.py:876 IOU "R1" [f9b76d55-6850-4935-8f45-21e906e87c04]: number of Serial adapters changed to 2
2021-06-02 20:15:42 INFO iou_vm.py:316 IOU "R1" [f9b76d55-6850-4935-8f45-21e906e87c04]: NVRAM updated from 128KB to 64KB
2021-06-02 20:15:42 INFO iou_vm.py:848 IOU "R1" [f9b76d55-6850-4935-8f45-21e906e87c04]: number of Ethernet adapters changed to 3
2021-06-02 20:15:42 INFO iou_vm.py:145 IOU "R1" [f9b76d55-6850-4935-8f45-21e906e87c04]: IOU image updated to "/home/saad/GNS3/images/IOU/i86bi-linux-l3-adventerprisek9-ms.155-2.T.bin"
2021-06-02 20:15:42 INFO iou_vm.py:876 IOU "R1" [f9b76d55-6850-4935-8f45-21e906e87c04]: number of Serial adapters changed to 1
2021-06-02 20:15:42 INFO web_log.py:206 ::1 [02/Jun/2021:15:15:42 +0000] "POST /v2/compute/projects/2a56fee1-bb43-4cda-a5f5-1ab3c90677e4/iou/nodes HTTP/1.1" 201 881 "-" "Python/3.9 aiohttp/3.7.4.post0"
2021-06-02 20:15:42 INFO iou_vm.py:73 IOU "R2" [49fc733e-16b0-4831-a5bb-47e90526e662]: assigned with application ID 2
2021-06-02 20:15:42 INFO iou_vm.py:848 IOU "R2" [49fc733e-16b0-4831-a5bb-47e90526e662]: number of Ethernet adapters changed to 2
2021-06-02 20:15:42 INFO iou_vm.py:876 IOU "R2" [49fc733e-16b0-4831-a5bb-47e90526e662]: number of Serial adapters changed to 2
2021-06-02 20:15:42 INFO iou_vm.py:316 IOU "R2" [49fc733e-16b0-4831-a5bb-47e90526e662]: NVRAM updated from 128KB to 64KB
2021-06-02 20:15:42 INFO iou_vm.py:848 IOU "R2" [49fc733e-16b0-4831-a5bb-47e90526e662]: number of Ethernet adapters changed to 3
2021-06-02 20:15:42 INFO iou_vm.py:145 IOU "R2" [49fc733e-16b0-4831-a5bb-47e90526e662]: IOU image updated to "/home/saad/GNS3/images/IOU/i86bi-linux-l3-adventerprisek9-ms.155-2.T.bin"
2021-06-02 20:15:42 INFO iou_vm.py:876 IOU "R2" [49fc733e-16b0-4831-a5bb-47e90526e662]: number of Serial adapters changed to 1
2021-06-02 20:15:42 INFO web_log.py:206 127.0.0.1 [02/Jun/2021:15:15:42 +0000] "POST /v2/compute/projects/2a56fee1-bb43-4cda-a5f5-1ab3c90677e4/iou/nodes HTTP/1.1" 201 881 "-" "Python/3.9 aiohttp/3.7.4.post0"
2021-06-02 20:15:42 INFO iou_vm.py:73 IOU "S1" [95ad63d1-5f53-4794-8366-2a7900692b25]: assigned with application ID 3
2021-06-02 20:15:42 INFO iou_vm.py:848 IOU "S1" [95ad63d1-5f53-4794-8366-2a7900692b25]: number of Ethernet adapters changed to 2
2021-06-02 20:15:42 INFO iou_vm.py:876 IOU "S1" [95ad63d1-5f53-4794-8366-2a7900692b25]: number of Serial adapters changed to 2
2021-06-02 20:15:42 INFO iou_vm.py:316 IOU "S1" [95ad63d1-5f53-4794-8366-2a7900692b25]: NVRAM updated from 128KB to 16KB
2021-06-02 20:15:42 INFO iou_vm.py:848 IOU "S1" [95ad63d1-5f53-4794-8366-2a7900692b25]: number of Ethernet adapters changed to 4
2021-06-02 20:15:42 INFO iou_vm.py:145 IOU "S1" [95ad63d1-5f53-4794-8366-2a7900692b25]: IOU image updated to "/home/saad/GNS3/images/IOU/i86bi-linux-l2-adventerprisek9-15.1a.bin"
2021-06-02 20:15:42 INFO iou_vm.py:876 IOU "S1" [95ad63d1-5f53-4794-8366-2a7900692b25]: number of Serial adapters changed to 0
2021-06-02 20:15:42 INFO web_log.py:206 ::1 [02/Jun/2021:15:15:42 +0000] "POST /v2/compute/projects/2a56fee1-bb43-4cda-a5f5-1ab3c90677e4/iou/nodes HTTP/1.1" 201 876 "-" "Python/3.9 aiohttp/3.7.4.post0"
2021-06-02 20:15:42 INFO docker_vm.py:1058 Docker container "debian-1" [46701752-b447-49c3-9e1f-6828f9b97d97]: number of Ethernet adapters changed to 1
2021-06-02 20:15:42 INFO docker_vm.py:406 Docker container 'debian-1' [46701752-b447-49c3-9e1f-6828f9b97d97] created
2021-06-02 20:15:42 INFO web_log.py:206 127.0.0.1 [02/Jun/2021:15:15:42 +0000] "POST /v2/compute/projects/2a56fee1-bb43-4cda-a5f5-1ab3c90677e4/docker/nodes HTTP/1.1" 201 935 "-" "Python/3.9 aiohttp/3.7.4.post0"
2021-06-02 20:15:42 INFO web_log.py:206 ::1 [02/Jun/2021:15:15:42 +0000] "POST /v2/compute/projects/2a56fee1-bb43-4cda-a5f5-1ab3c90677e4/ports/udp HTTP/1.1" 201 243 "-" "Python/3.9 aiohttp/3.7.4.post0"
2021-06-02 20:15:42 INFO web_log.py:206 127.0.0.1 [02/Jun/2021:15:15:42 +0000] "POST /v2/compute/projects/2a56fee1-bb43-4cda-a5f5-1ab3c90677e4/ports/udp HTTP/1.1" 201 243 "-" "Python/3.9 aiohttp/3.7.4.post0"
2021-06-02 20:15:42 INFO iou_vm.py:902 IOU "S1" [95ad63d1-5f53-4794-8366-2a7900692b25]: NIO UDP added to 0/0
2021-06-02 20:15:42 INFO web_log.py:206 ::1 [02/Jun/2021:15:15:42 +0000] "POST /v2/compute/projects/2a56fee1-bb43-4cda-a5f5-1ab3c90677e4/iou/nodes/95ad63d1-5f53-4794-8366-2a7900692b25/adapters/0/ports/0/nio HTTP/1.1" 201 377 "-" "Python/3.9 aiohttp/3.7.4.post0"
2021-06-02 20:15:42 INFO docker_vm.py:960 Docker container 'debian-1' [46701752-b447-49c3-9e1f-6828f9b97d97]: NIO UDP added to adapter 0
2021-06-02 20:15:42 INFO web_log.py:206 127.0.0.1 [02/Jun/2021:15:15:42 +0000] "POST /v2/compute/projects/2a56fee1-bb43-4cda-a5f5-1ab3c90677e4/docker/nodes/46701752-b447-49c3-9e1f-6828f9b97d97/adapters/0/ports/0/nio HTTP/1.1" 201 380 "-" "Python/3.9 aiohttp/3.7.4.post0"
2021-06-02 20:15:42 INFO web_log.py:206 ::1 [02/Jun/2021:15:15:42 +0000] "POST /v2/compute/projects/2a56fee1-bb43-4cda-a5f5-1ab3c90677e4/ports/udp HTTP/1.1" 201 243 "-" "Python/3.9 aiohttp/3.7.4.post0"
2021-06-02 20:15:42 INFO web_log.py:206 127.0.0.1 [02/Jun/2021:15:15:42 +0000] "POST /v2/compute/projects/2a56fee1-bb43-4cda-a5f5-1ab3c90677e4/ports/udp HTTP/1.1" 201 243 "-" "Python/3.9 aiohttp/3.7.4.post0"
2021-06-02 20:15:42 INFO iou_vm.py:902 IOU "S1" [95ad63d1-5f53-4794-8366-2a7900692b25]: NIO UDP added to 0/1
2021-06-02 20:15:42 INFO web_log.py:206 ::1 [02/Jun/2021:15:15:42 +0000] "POST /v2/compute/projects/2a56fee1-bb43-4cda-a5f5-1ab3c90677e4/iou/nodes/95ad63d1-5f53-4794-8366-2a7900692b25/adapters/0/ports/1/nio HTTP/1.1" 201 377 "-" "Python/3.9 aiohttp/3.7.4.post0"
2021-06-02 20:15:43 INFO iou_vm.py:902 IOU "R2" [49fc733e-16b0-4831-a5bb-47e90526e662]: NIO UDP added to 0/0
2021-06-02 20:15:43 INFO web_log.py:206 127.0.0.1 [02/Jun/2021:15:15:42 +0000] "POST /v2/compute/projects/2a56fee1-bb43-4cda-a5f5-1ab3c90677e4/iou/nodes/49fc733e-16b0-4831-a5bb-47e90526e662/adapters/0/ports/0/nio HTTP/1.1" 201 377 "-" "Python/3.9 aiohttp/3.7.4.post0"
2021-06-02 20:15:43 INFO web_log.py:206 ::1 [02/Jun/2021:15:15:43 +0000] "POST /v2/compute/projects/2a56fee1-bb43-4cda-a5f5-1ab3c90677e4/ports/udp HTTP/1.1" 201 243 "-" "Python/3.9 aiohttp/3.7.4.post0"
2021-06-02 20:15:43 INFO web_log.py:206 127.0.0.1 [02/Jun/2021:15:15:43 +0000] "POST /v2/compute/projects/2a56fee1-bb43-4cda-a5f5-1ab3c90677e4/ports/udp HTTP/1.1" 201 243 "-" "Python/3.9 aiohttp/3.7.4.post0"
2021-06-02 20:15:43 INFO iou_vm.py:902 IOU "R2" [49fc733e-16b0-4831-a5bb-47e90526e662]: NIO UDP added to 3/0
2021-06-02 20:15:43 INFO web_log.py:206 ::1 [02/Jun/2021:15:15:43 +0000] "POST /v2/compute/projects/2a56fee1-bb43-4cda-a5f5-1ab3c90677e4/iou/nodes/49fc733e-16b0-4831-a5bb-47e90526e662/adapters/3/ports/0/nio HTTP/1.1" 201 377 "-" "Python/3.9 aiohttp/3.7.4.post0"
2021-06-02 20:15:43 INFO iou_vm.py:902 IOU "R1" [f9b76d55-6850-4935-8f45-21e906e87c04]: NIO UDP added to 3/0
2021-06-02 20:15:43 INFO web_log.py:206 127.0.0.1 [02/Jun/2021:15:15:43 +0000] "POST /v2/compute/projects/2a56fee1-bb43-4cda-a5f5-1ab3c90677e4/iou/nodes/f9b76d55-6850-4935-8f45-21e906e87c04/adapters/3/ports/0/nio HTTP/1.1" 201 377 "-" "Python/3.9 aiohttp/3.7.4.post0"
2021-06-02 20:15:43 INFO web_log.py:206 ::1 [02/Jun/2021:15:15:43 +0000] "POST /v2/compute/projects/2a56fee1-bb43-4cda-a5f5-1ab3c90677e4/ports/udp HTTP/1.1" 201 243 "-" "Python/3.9 aiohttp/3.7.4.post0"
2021-06-02 20:15:43 INFO web_log.py:206 127.0.0.1 [02/Jun/2021:15:15:43 +0000] "POST /v2/compute/projects/2a56fee1-bb43-4cda-a5f5-1ab3c90677e4/ports/udp HTTP/1.1" 201 243 "-" "Python/3.9 aiohttp/3.7.4.post0"
2021-06-02 20:15:43 INFO iou_vm.py:902 IOU "R1" [f9b76d55-6850-4935-8f45-21e906e87c04]: NIO UDP added to 0/0
2021-06-02 20:15:43 INFO web_log.py:206 ::1 [02/Jun/2021:15:15:43 +0000] "POST /v2/compute/projects/2a56fee1-bb43-4cda-a5f5-1ab3c90677e4/iou/nodes/f9b76d55-6850-4935-8f45-21e906e87c04/adapters/0/ports/0/nio HTTP/1.1" 201 377 "-" "Python/3.9 aiohttp/3.7.4.post0"
2021-06-02 20:15:43 INFO cloud.py:396 Cloud "NAT1" [9c32e5f4-9a89-4939-82c0-22ddd64553a7]: NIO NIO UDP bound to port 0
2021-06-02 20:15:43 INFO web_log.py:206 127.0.0.1 [02/Jun/2021:15:15:43 +0000] "POST /v2/compute/projects/2a56fee1-bb43-4cda-a5f5-1ab3c90677e4/nat/nodes/9c32e5f4-9a89-4939-82c0-22ddd64553a7/adapters/0/ports/0/nio HTTP/1.1" 201 377 "-" "Python/3.9 aiohttp/3.7.4.post0"
2021-06-02 20:15:43 INFO web_log.py:206 ::1 [02/Jun/2021:15:15:42 +0000] "POST /v2/projects/load HTTP/1.1" 201 763 "-" "GNS3 QT Client v2.2.21"
2021-06-02 20:15:43 INFO project_handler.py:271 New client has connected to the notification stream for project ID '2a56fee1-bb43-4cda-a5f5-1ab3c90677e4' (WebSocket method)
2021-06-02 20:15:43 INFO web_log.py:206 ::1 [02/Jun/2021:15:15:43 +0000] "GET /v2/projects/2a56fee1-bb43-4cda-a5f5-1ab3c90677e4/nodes HTTP/1.1" 200 24694 "-" "GNS3 QT Client v2.2.21"
2021-06-02 20:15:43 INFO web_log.py:206 ::1 [02/Jun/2021:15:15:43 +0000] "GET /v2/projects/2a56fee1-bb43-4cda-a5f5-1ab3c90677e4/links HTTP/1.1" 200 5422 "-" "GNS3 QT Client v2.2.21"
2021-06-02 20:15:43 INFO web_log.py:206 ::1 [02/Jun/2021:15:15:43 +0000] "GET /v2/projects/2a56fee1-bb43-4cda-a5f5-1ab3c90677e4/drawings HTTP/1.1" 200 619 "-" "GNS3 QT Client v2.2.21"
2021-06-02 20:15:46 INFO web_log.py:206 127.0.0.1 [02/Jun/2021:15:15:46 +0000] "POST /v2/compute/projects/2a56fee1-bb43-4cda-a5f5-1ab3c90677e4/nat/nodes/9c32e5f4-9a89-4939-82c0-22ddd64553a7/start HTTP/1.1" 204 225 "-" "Python/3.9 aiohttp/3.7.4.post0"
2021-06-02 20:15:46 INFO iou_vm.py:414 Checking IOU license in '/tmp/tmpkxfvzery/iourc'
2021-06-02 20:15:46 INFO iou_vm.py:414 Checking IOU license in '/tmp/tmptvryynpq/iourc'
2021-06-02 20:15:46 INFO base_node.py:655 Starting new uBridge hypervisor localhost:33467
2021-06-02 20:15:46 INFO base_node.py:655 Starting new uBridge hypervisor localhost:38041
2021-06-02 20:15:46 INFO hypervisor.py:169 starting ubridge: ['/sbin/ubridge', '-H', 'localhost:33467']
2021-06-02 20:15:46 INFO hypervisor.py:171 logging to /home/saad/GNS3/projects/lab_for_programming/project-files/iou/f9b76d55-6850-4935-8f45-21e906e87c04/ubridge.log
2021-06-02 20:15:46 INFO hypervisor.py:179 ubridge started PID=43391
2021-06-02 20:15:46 INFO base_node.py:658 Hypervisor localhost:33467 has successfully started
2021-06-02 20:15:46 INFO iou_vm.py:414 Checking IOU license in '/tmp/tmp0t4bsac0/iourc'
2021-06-02 20:15:46 INFO hypervisor.py:169 starting ubridge: ['/sbin/ubridge', '-H', 'localhost:38041']
2021-06-02 20:15:46 INFO hypervisor.py:171 logging to /home/saad/GNS3/projects/lab_for_programming/project-files/iou/49fc733e-16b0-4831-a5bb-47e90526e662/ubridge.log
2021-06-02 20:15:46 INFO hypervisor.py:179 ubridge started PID=43395
2021-06-02 20:15:46 INFO base_node.py:658 Hypervisor localhost:38041 has successfully started
2021-06-02 20:15:46 INFO base_node.py:655 Starting new uBridge hypervisor localhost:46505
2021-06-02 20:15:46 INFO hypervisor.py:169 starting ubridge: ['/sbin/ubridge', '-H', 'localhost:46505']
2021-06-02 20:15:46 INFO hypervisor.py:171 logging to /home/saad/GNS3/projects/lab_for_programming/project-files/iou/95ad63d1-5f53-4794-8366-2a7900692b25/ubridge.log
2021-06-02 20:15:46 INFO hypervisor.py:179 ubridge started PID=43399
2021-06-02 20:15:46 INFO base_node.py:658 Hypervisor localhost:46505 has successfully started
2021-06-02 20:15:46 INFO ubridge_hypervisor.py:83 Connected to uBridge hypervisor on localhost:33467 after 0.1056 seconds
2021-06-02 20:15:46 INFO ubridge_hypervisor.py:83 Connected to uBridge hypervisor on localhost:38041 after 0.1055 seconds
2021-06-02 20:15:46 INFO ubridge_hypervisor.py:83 Connected to uBridge hypervisor on localhost:46505 after 0.1049 seconds
2021-06-02 20:15:46 INFO iou_vm.py:753 IOU R1 [id=f9b76d55-6850-4935-8f45-21e906e87c04]: NETMAP file created
2021-06-02 20:15:46 INFO iou_vm.py:551 Starting IOU: ['/home/saad/GNS3/images/IOU/i86bi-linux-l3-adventerprisek9-ms.155-2.T.bin', '-e', '3', '-s', '1', '1']
2021-06-02 20:15:47 INFO iou_vm.py:753 IOU R2 [id=49fc733e-16b0-4831-a5bb-47e90526e662]: NETMAP file created
2021-06-02 20:15:47 INFO iou_vm.py:753 IOU S1 [id=95ad63d1-5f53-4794-8366-2a7900692b25]: NETMAP file created
2021-06-02 20:15:47 INFO iou_vm.py:560 IOU instance f9b76d55-6850-4935-8f45-21e906e87c04 started PID=43410
2021-06-02 20:15:47 INFO iou_vm.py:551 Starting IOU: ['/home/saad/GNS3/images/IOU/i86bi-linux-l3-adventerprisek9-ms.155-2.T.bin', '-e', '3', '-s', '1', '2']
2021-06-02 20:15:47 INFO iou_vm.py:551 Starting IOU: ['/home/saad/GNS3/images/IOU/i86bi-linux-l2-adventerprisek9-15.1a.bin', '-e', '4', '-s', '0', '3']
2021-06-02 20:15:47 INFO iou_vm.py:560 IOU instance 49fc733e-16b0-4831-a5bb-47e90526e662 started PID=43416
2021-06-02 20:15:47 INFO iou_vm.py:560 IOU instance 95ad63d1-5f53-4794-8366-2a7900692b25 started PID=43418
2021-06-02 20:15:47 INFO web_log.py:206 ::1 [02/Jun/2021:15:15:46 +0000] "POST /v2/compute/projects/2a56fee1-bb43-4cda-a5f5-1ab3c90677e4/iou/nodes/f9b76d55-6850-4935-8f45-21e906e87c04/start HTTP/1.1" 200 976 "-" "Python/3.9 aiohttp/3.7.4.post0"
2021-06-02 20:15:47 INFO web_log.py:206 ::1 [02/Jun/2021:15:15:46 +0000] "POST /v2/compute/projects/2a56fee1-bb43-4cda-a5f5-1ab3c90677e4/iou/nodes/49fc733e-16b0-4831-a5bb-47e90526e662/start HTTP/1.1" 200 976 "-" "Python/3.9 aiohttp/3.7.4.post0"
2021-06-02 20:15:47 INFO web_log.py:206 127.0.0.1 [02/Jun/2021:15:15:46 +0000] "POST /v2/compute/projects/2a56fee1-bb43-4cda-a5f5-1ab3c90677e4/iou/nodes/95ad63d1-5f53-4794-8366-2a7900692b25/start HTTP/1.1" 200 966 "-" "Python/3.9 aiohttp/3.7.4.post0"
2021-06-02 20:15:47 INFO base_node.py:655 Starting new uBridge hypervisor localhost:46015
2021-06-02 20:15:47 INFO hypervisor.py:169 starting ubridge: ['/sbin/ubridge', '-H', 'localhost:46015']
2021-06-02 20:15:47 INFO hypervisor.py:171 logging to /home/saad/GNS3/projects/lab_for_programming/project-files/docker/46701752-b447-49c3-9e1f-6828f9b97d97/ubridge.log
2021-06-02 20:15:47 INFO hypervisor.py:179 ubridge started PID=43492
2021-06-02 20:15:47 INFO base_node.py:658 Hypervisor localhost:46015 has successfully started
2021-06-02 20:15:47 INFO ubridge_hypervisor.py:83 Connected to uBridge hypervisor on localhost:46015 after 0.1046 seconds
2021-06-02 20:15:47 INFO docker_vm.py:499 Docker container 'debian-1' [debian:latest] started listen for telnet on 5003
2021-06-02 20:15:48 INFO web_log.py:206 ::1 [02/Jun/2021:15:15:47 +0000] "POST /v2/compute/projects/2a56fee1-bb43-4cda-a5f5-1ab3c90677e4/docker/nodes/46701752-b447-49c3-9e1f-6828f9b97d97/start HTTP/1.1" 204 228 "-" "Python/3.9 aiohttp/3.7.4.post0"
2021-06-02 20:15:48 INFO web_log.py:206 ::1 [02/Jun/2021:15:15:46 +0000] "POST /v2/projects/2a56fee1-bb43-4cda-a5f5-1ab3c90677e4/nodes/start HTTP/1.1" 204 203 "-" "GNS3 QT Client v2.2.21"
2021-06-02 20:15:48 INFO web_log.py:206 ::1 [02/Jun/2021:15:15:48 +0000] "GET /v2/computes HTTP/1.1" 200 1023 "-" "GNS3 QT Client v2.2.21"
2021-06-02 20:16:40 INFO web_log.py:206 ::1 [02/Jun/2021:15:16:40 +0000] "GET /v2/computes HTTP/1.1" 200 1023 "-" "GNS3 QT Client v2.2.21"
2021-06-02 20:21:05 INFO web_log.py:206 ::1 [02/Jun/2021:15:21:05 +0000] "GET /v2/computes HTTP/1.1" 200 1022 "-" "GNS3 QT Client v2.2.21"
2021-06-02 20:27:31 INFO web_log.py:206 ::1 [02/Jun/2021:15:27:31 +0000] "GET /v2/computes HTTP/1.1" 200 1022 "-" "GNS3 QT Client v2.2.21"
2021-06-02 20:28:15 INFO docker_vm.py:810 Docker container 'debian-1' [debian:latest] paused
2021-06-02 20:28:15 INFO web_log.py:206 ::1 [02/Jun/2021:15:28:15 +0000] "POST /v2/compute/projects/2a56fee1-bb43-4cda-a5f5-1ab3c90677e4/docker/nodes/46701752-b447-49c3-9e1f-6828f9b97d97/suspend HTTP/1.1" 204 230 "-" "Python/3.9 aiohttp/3.7.4.post0"
2021-06-02 20:28:15 INFO web_log.py:206 ::1 [02/Jun/2021:15:28:15 +0000] "POST /v2/projects/2a56fee1-bb43-4cda-a5f5-1ab3c90677e4/nodes/46701752-b447-49c3-9e1f-6828f9b97d97/suspend HTTP/1.1" 200 1987 "-" "GNS3 QT Client v2.2.21"
2021-06-02 20:28:17 INFO base_node.py:669 Stopping uBridge hypervisor localhost:46015
2021-06-02 20:28:17 INFO hypervisor.py:208 Stopping uBridge process PID=43492
2021-06-02 20:28:18 INFO docker_vm.py:819 Docker container 'debian-1' [debian:latest] unpaused
2021-06-02 20:28:18 INFO docker_vm.py:533 Docker container 'debian-1' fix ownership, state = running
2021-06-02 20:28:18 INFO web_log.py:206 ::1 [02/Jun/2021:15:28:18 +0000] "GET /v2/computes HTTP/1.1" 200 1022 "-" "GNS3 QT Client v2.2.21"
2021-06-02 20:28:18 INFO docker_vm.py:793 Docker container 'debian-1' [debian:latest] stopped
2021-06-02 20:28:18 INFO web_log.py:206 127.0.0.1 [02/Jun/2021:15:28:17 +0000] "POST /v2/compute/projects/2a56fee1-bb43-4cda-a5f5-1ab3c90677e4/docker/nodes/46701752-b447-49c3-9e1f-6828f9b97d97/stop HTTP/1.1" 204 227 "-" "Python/3.9 aiohttp/3.7.4.post0"
2021-06-02 20:28:18 INFO web_log.py:206 ::1 [02/Jun/2021:15:28:17 +0000] "POST /v2/projects/2a56fee1-bb43-4cda-a5f5-1ab3c90677e4/nodes/46701752-b447-49c3-9e1f-6828f9b97d97/stop HTTP/1.1" 200 1984 "-" "GNS3 QT Client v2.2.21"
2021-06-02 20:28:20 INFO web_log.py:206 ::1 [02/Jun/2021:15:28:20 +0000] "GET /v2/projects/2a56fee1-bb43-4cda-a5f5-1ab3c90677e4/nodes/46701752-b447-49c3-9e1f-6828f9b97d97/files/etc/network/interfaces HTTP/1.1" 200 574 "-" "GNS3 QT Client v2.2.21"
2021-06-02 20:28:24 INFO web_log.py:206 127.0.0.1 [02/Jun/2021:15:28:24 +0000] "POST /v2/compute/projects/2a56fee1-bb43-4cda-a5f5-1ab3c90677e4/files/project-files/docker/46701752-b447-49c3-9e1f-6828f9b97d97/etc/network/interfaces HTTP/1.1" 200 226 "-" "Python/3.9 aiohttp/3.7.4.post0"
2021-06-02 20:28:24 INFO web_log.py:206 ::1 [02/Jun/2021:15:28:24 +0000] "POST /v2/projects/2a56fee1-bb43-4cda-a5f5-1ab3c90677e4/nodes/46701752-b447-49c3-9e1f-6828f9b97d97/files/etc/network/interfaces HTTP/1.1" 201 239 "-" "GNS3 QT Client v2.2.21"
2021-06-02 20:28:26 INFO base_node.py:655 Starting new uBridge hypervisor localhost:43305
2021-06-02 20:28:26 INFO hypervisor.py:169 starting ubridge: ['/sbin/ubridge', '-H', 'localhost:43305']
2021-06-02 20:28:26 INFO hypervisor.py:171 logging to /home/saad/GNS3/projects/lab_for_programming/project-files/docker/46701752-b447-49c3-9e1f-6828f9b97d97/ubridge.log
2021-06-02 20:28:26 INFO hypervisor.py:179 ubridge started PID=49615
2021-06-02 20:28:26 INFO base_node.py:658 Hypervisor localhost:43305 has successfully started
2021-06-02 20:28:26 INFO ubridge_hypervisor.py:83 Connected to uBridge hypervisor on localhost:43305 after 0.1033 seconds
2021-06-02 20:28:27 INFO docker_vm.py:499 Docker container 'debian-1' [debian:latest] started listen for telnet on 5003
2021-06-02 20:28:27 INFO web_log.py:206 ::1 [02/Jun/2021:15:28:26 +0000] "POST /v2/compute/projects/2a56fee1-bb43-4cda-a5f5-1ab3c90677e4/docker/nodes/46701752-b447-49c3-9e1f-6828f9b97d97/start HTTP/1.1" 204 228 "-" "Python/3.9 aiohttp/3.7.4.post0"
2021-06-02 20:28:27 INFO web_log.py:206 ::1 [02/Jun/2021:15:28:26 +0000] "POST /v2/projects/2a56fee1-bb43-4cda-a5f5-1ab3c90677e4/nodes/46701752-b447-49c3-9e1f-6828f9b97d97/start HTTP/1.1" 200 1985 "-" "GNS3 QT Client v2.2.21"
2021-06-02 20:29:01 INFO web_log.py:206 ::1 [02/Jun/2021:15:29:01 +0000] "GET /v2/computes HTTP/1.1" 200 1022 "-" "GNS3 QT Client v2.2.21"
2021-06-02 20:35:53 INFO web_log.py:206 ::1 [02/Jun/2021:15:35:53 +0000] "GET /v2/computes HTTP/1.1" 200 1023 "-" "GNS3 QT Client v2.2.21"
2021-06-02 20:41:30 INFO web_log.py:206 ::1 [02/Jun/2021:15:41:30 +0000] "GET /v2/computes HTTP/1.1" 200 1022 "-" "GNS3 QT Client v2.2.21"
2021-06-02 20:42:34 INFO base_node.py:669 Stopping uBridge hypervisor localhost:38041
2021-06-02 20:42:34 INFO hypervisor.py:208 Stopping uBridge process PID=43395
2021-06-02 20:42:34 INFO web_log.py:206 127.0.0.1 [02/Jun/2021:15:42:34 +0000] "POST /v2/compute/projects/2a56fee1-bb43-4cda-a5f5-1ab3c90677e4/nat/nodes/9c32e5f4-9a89-4939-82c0-22ddd64553a7/stop HTTP/1.1" 204 224 "-" "Python/3.9 aiohttp/3.7.4.post0"
2021-06-02 20:42:34 INFO base_node.py:669 Stopping uBridge hypervisor localhost:33467
2021-06-02 20:42:34 INFO hypervisor.py:208 Stopping uBridge process PID=43391
2021-06-02 20:42:34 INFO base_node.py:669 Stopping uBridge hypervisor localhost:46505
2021-06-02 20:42:34 INFO hypervisor.py:208 Stopping uBridge process PID=43399
2021-06-02 20:42:34 INFO iou_vm.py:698 Stopping IOU process for IOU VM "R2" PID=43416
2021-06-02 20:42:34 INFO iou_vm.py:698 Stopping IOU process for IOU VM "R2" PID=43416
2021-06-02 20:42:34 INFO iou_vm.py:1268 saving startup-config to /home/saad/GNS3/projects/lab_for_programming/project-files/iou/49fc733e-16b0-4831-a5bb-47e90526e662/startup-config.cfg
2021-06-02 20:42:34 INFO iou_vm.py:1278 saving private-config to /home/saad/GNS3/projects/lab_for_programming/project-files/iou/49fc733e-16b0-4831-a5bb-47e90526e662/private-config.cfg
2021-06-02 20:42:34 INFO web_log.py:206 ::1 [02/Jun/2021:15:42:34 +0000] "POST /v2/compute/projects/2a56fee1-bb43-4cda-a5f5-1ab3c90677e4/iou/nodes/49fc733e-16b0-4831-a5bb-47e90526e662/stop HTTP/1.1" 204 224 "-" "Python/3.9 aiohttp/3.7.4.post0"
2021-06-02 20:42:34 INFO iou_vm.py:698 Stopping IOU process for IOU VM "R1" PID=43410
2021-06-02 20:42:34 INFO base_node.py:669 Stopping uBridge hypervisor localhost:43305
2021-06-02 20:42:34 INFO hypervisor.py:208 Stopping uBridge process PID=49615
2021-06-02 20:42:34 INFO iou_vm.py:698 Stopping IOU process for IOU VM "R1" PID=43410
2021-06-02 20:42:34 INFO iou_vm.py:1268 saving startup-config to /home/saad/GNS3/projects/lab_for_programming/project-files/iou/f9b76d55-6850-4935-8f45-21e906e87c04/startup-config.cfg
2021-06-02 20:42:34 INFO iou_vm.py:1278 saving private-config to /home/saad/GNS3/projects/lab_for_programming/project-files/iou/f9b76d55-6850-4935-8f45-21e906e87c04/private-config.cfg
2021-06-02 20:42:34 INFO web_log.py:206 ::1 [02/Jun/2021:15:42:34 +0000] "POST /v2/compute/projects/2a56fee1-bb43-4cda-a5f5-1ab3c90677e4/iou/nodes/f9b76d55-6850-4935-8f45-21e906e87c04/stop HTTP/1.1" 204 224 "-" "Python/3.9 aiohttp/3.7.4.post0"
2021-06-02 20:42:34 INFO iou_vm.py:698 Stopping IOU process for IOU VM "S1" PID=43418
2021-06-02 20:42:34 INFO iou_vm.py:698 Stopping IOU process for IOU VM "S1" PID=43418
2021-06-02 20:42:34 INFO iou_vm.py:1268 saving startup-config to /home/saad/GNS3/projects/lab_for_programming/project-files/iou/95ad63d1-5f53-4794-8366-2a7900692b25/startup-config.cfg
2021-06-02 20:42:34 INFO web_log.py:206 127.0.0.1 [02/Jun/2021:15:42:34 +0000] "POST /v2/compute/projects/2a56fee1-bb43-4cda-a5f5-1ab3c90677e4/iou/nodes/95ad63d1-5f53-4794-8366-2a7900692b25/stop HTTP/1.1" 204 224 "-" "Python/3.9 aiohttp/3.7.4.post0"
2021-06-02 20:42:34 INFO docker_vm.py:533 Docker container 'debian-1' fix ownership, state = running
2021-06-02 20:42:34 INFO docker_vm.py:793 Docker container 'debian-1' [debian:latest] stopped
2021-06-02 20:42:34 INFO web_log.py:206 ::1 [02/Jun/2021:15:42:34 +0000] "POST /v2/compute/projects/2a56fee1-bb43-4cda-a5f5-1ab3c90677e4/docker/nodes/46701752-b447-49c3-9e1f-6828f9b97d97/stop HTTP/1.1" 204 227 "-" "Python/3.9 aiohttp/3.7.4.post0"
2021-06-02 20:42:34 INFO web_log.py:206 ::1 [02/Jun/2021:15:42:34 +0000] "POST /v2/projects/2a56fee1-bb43-4cda-a5f5-1ab3c90677e4/nodes/stop HTTP/1.1" 204 202 "-" "GNS3 QT Client v2.2.21"
2021-06-02 20:42:35 INFO web_log.py:206 ::1 [02/Jun/2021:15:42:35 +0000] "GET /v2/computes HTTP/1.1" 200 1022 "-" "GNS3 QT Client v2.2.21"
2021-06-02 20:42:36 INFO notification_handler.py:82 Client has disconnected from controller WebSocket
2021-06-02 20:42:36 INFO server_handler.py:53 Start shutting down the server
2021-06-02 20:42:36 INFO iou_vm.py:1268 saving startup-config to /home/saad/GNS3/projects/lab_for_programming/project-files/iou/49fc733e-16b0-4831-a5bb-47e90526e662/startup-config.cfg
2021-06-02 20:42:36 INFO iou_vm.py:1278 saving private-config to /home/saad/GNS3/projects/lab_for_programming/project-files/iou/49fc733e-16b0-4831-a5bb-47e90526e662/private-config.cfg
2021-06-02 20:42:36 INFO web_log.py:206 127.0.0.1 [02/Jun/2021:15:42:36 +0000] "POST /v2/compute/projects/2a56fee1-bb43-4cda-a5f5-1ab3c90677e4/iou/nodes/49fc733e-16b0-4831-a5bb-47e90526e662/stop HTTP/1.1" 204 224 "-" "Python/3.9 aiohttp/3.7.4.post0"
2021-06-02 20:42:36 INFO web_log.py:206 ::1 [02/Jun/2021:15:42:36 +0000] "POST /v2/compute/projects/2a56fee1-bb43-4cda-a5f5-1ab3c90677e4/nat/nodes/9c32e5f4-9a89-4939-82c0-22ddd64553a7/stop HTTP/1.1" 204 224 "-" "Python/3.9 aiohttp/3.7.4.post0"
2021-06-02 20:42:36 INFO iou_vm.py:1268 saving startup-config to /home/saad/GNS3/projects/lab_for_programming/project-files/iou/f9b76d55-6850-4935-8f45-21e906e87c04/startup-config.cfg
2021-06-02 20:42:36 INFO iou_vm.py:1278 saving private-config to /home/saad/GNS3/projects/lab_for_programming/project-files/iou/f9b76d55-6850-4935-8f45-21e906e87c04/private-config.cfg
2021-06-02 20:42:36 INFO web_log.py:206 127.0.0.1 [02/Jun/2021:15:42:36 +0000] "POST /v2/compute/projects/2a56fee1-bb43-4cda-a5f5-1ab3c90677e4/iou/nodes/f9b76d55-6850-4935-8f45-21e906e87c04/stop HTTP/1.1" 204 224 "-" "Python/3.9 aiohttp/3.7.4.post0"
2021-06-02 20:42:36 INFO iou_vm.py:1268 saving startup-config to /home/saad/GNS3/projects/lab_for_programming/project-files/iou/95ad63d1-5f53-4794-8366-2a7900692b25/startup-config.cfg
2021-06-02 20:42:36 INFO web_log.py:206 127.0.0.1 [02/Jun/2021:15:42:36 +0000] "POST /v2/compute/projects/2a56fee1-bb43-4cda-a5f5-1ab3c90677e4/iou/nodes/95ad63d1-5f53-4794-8366-2a7900692b25/stop HTTP/1.1" 204 224 "-" "Python/3.9 aiohttp/3.7.4.post0"
2021-06-02 20:42:36 INFO web_log.py:206 ::1 [02/Jun/2021:15:42:36 +0000] "POST /v2/compute/projects/2a56fee1-bb43-4cda-a5f5-1ab3c90677e4/docker/nodes/46701752-b447-49c3-9e1f-6828f9b97d97/stop HTTP/1.1" 204 227 "-" "Python/3.9 aiohttp/3.7.4.post0"
2021-06-02 20:42:36 INFO base_node.py:331 IOU: 'S1' [95ad63d1-5f53-4794-8366-2a7900692b25]: is closing
2021-06-02 20:42:36 INFO iou_vm.py:1268 saving startup-config to /home/saad/GNS3/projects/lab_for_programming/project-files/iou/95ad63d1-5f53-4794-8366-2a7900692b25/startup-config.cfg
2021-06-02 20:42:36 INFO base_node.py:331 Docker: 'debian-1' [46701752-b447-49c3-9e1f-6828f9b97d97]: is closing
2021-06-02 20:42:36 INFO base_node.py:331 Builtin: 'NAT1' [9c32e5f4-9a89-4939-82c0-22ddd64553a7]: is closing
2021-06-02 20:42:36 INFO base_node.py:669 Stopping uBridge hypervisor localhost:35017
2021-06-02 20:42:36 INFO hypervisor.py:208 Stopping uBridge process PID=43277
2021-06-02 20:42:36 INFO base_node.py:331 IOU: 'R1' [f9b76d55-6850-4935-8f45-21e906e87c04]: is closing
2021-06-02 20:42:36 INFO iou_vm.py:1268 saving startup-config to /home/saad/GNS3/projects/lab_for_programming/project-files/iou/f9b76d55-6850-4935-8f45-21e906e87c04/startup-config.cfg
2021-06-02 20:42:36 INFO iou_vm.py:1278 saving private-config to /home/saad/GNS3/projects/lab_for_programming/project-files/iou/f9b76d55-6850-4935-8f45-21e906e87c04/private-config.cfg
2021-06-02 20:42:36 INFO base_node.py:331 IOU: 'R2' [49fc733e-16b0-4831-a5bb-47e90526e662]: is closing
2021-06-02 20:42:36 INFO iou_vm.py:1268 saving startup-config to /home/saad/GNS3/projects/lab_for_programming/project-files/iou/49fc733e-16b0-4831-a5bb-47e90526e662/startup-config.cfg
2021-06-02 20:42:36 INFO iou_vm.py:1278 saving private-config to /home/saad/GNS3/projects/lab_for_programming/project-files/iou/49fc733e-16b0-4831-a5bb-47e90526e662/private-config.cfg
2021-06-02 20:42:36 INFO docker_vm.py:872 Docker container 'debian-1' [debian:latest] removed
2021-06-02 20:42:36 INFO cloud.py:248 Cloud "NAT1" [9c32e5f4-9a89-4939-82c0-22ddd64553a7] has been closed
2021-06-02 20:42:36 INFO project.py:361 Project 2a56fee1-bb43-4cda-a5f5-1ab3c90677e4 with path '/home/saad/GNS3/projects/lab_for_programming' closed
2021-06-02 20:42:36 INFO web_log.py:206 ::1 [02/Jun/2021:15:42:36 +0000] "POST /v2/compute/projects/2a56fee1-bb43-4cda-a5f5-1ab3c90677e4/close HTTP/1.1" 204 205 "-" "Python/3.9 aiohttp/3.7.4.post0"
2021-06-02 20:42:36 INFO web_log.py:206 ::1 [02/Jun/2021:15:42:36 +0000] "POST /v2/shutdown HTTP/1.1" 201 194 "-" "GNS3 QT Client v2.2.21"
2021-06-02 20:42:36 INFO web_server.py:113 Closing 2 websocket connections...
2021-06-02 20:42:36 INFO compute.py:460 Connection closed to compute 'local' WebSocket 'http://localhost:3080/v2/compute/notifications/ws'
2021-06-02 20:42:36 INFO project_handler.py:280 Client has disconnected from notification stream for project ID '2a56fee1-bb43-4cda-a5f5-1ab3c90677e4' (WebSocket method)
2021-06-02 20:42:36 INFO notification_handler.py:59 Client has disconnected from compute WebSocket
2021-06-02 20:42:36 INFO __init__.py:138 Controller is stopping
2021-06-02 20:42:36 INFO run.py:272 Remove PID file /home/saad/.config/GNS3/2.2/gns3_server.pid
